daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SValue.scala

```diff
@@ -187,8 +187,13 @@ object SValue {
 
   object SRecord {
     def apply(id: Identifier, fields: ImmArray[Name], values: util.ArrayList[SValue]): SRecord = {
-      assert(fields.length == values.size)
-      val zipped = (fields.toSeq zip values.asScala).toList.reverse
```

Comment:
This `reverse` was left in by accident on the previous PR. It was just a development-time test to ensure nothing depended on the order any longer.